### PR TITLE
Add route to stream logs

### PR DESCRIPTION
**Pull request by nkatzman**
This PR relies on the k8s executor change in PR https://github.com/screwdriver-cd/executor-k8s/pull/8 

This route calls the buildModel.stream function with the buildId and reply object

The buildModel calls the executor.stream function passing in the two parameters

### CODE REVIEW

File: lib/buildmodel.js [hunk 1]
```diff
@@ -124,6 +124,19 @@ const BuildModel = class {
             });
         });
     }
+
+    /**
+     * Stream a build
+     * @method create
```
Comment (d2lam) [inline lib/buildmodel.js]: stream

File: lib/stream.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const joi = require('joi');
+const BuildModel = require('./buildmodel');
+const dataModel = require('screwdriver-data-model');
+const idModel = joi.reach(dataModel.build.base, 'id');
+
+module.exports = (datastore) => ({
+    method: 'GET',
+    path: '/builds/{id}/logs',
+    config: {
+        description: 'Get a single build',
```
Comment (d2lam) [inline lib/stream.js]: Get logs for a build

File: lib/stream.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        notes: 'Returns a build record',
```
Comment (d2lam) [inline lib/stream.js]: Return build logs

**Comment by d2lam** [comment]
Looks good. 👍  
Some small comments & need to update the k8s version when it's done.

File: lib/buildmodel.js [hunk 4]
```diff
@@ -124,6 +124,19 @@ const BuildModel = class {
...
+     * @param  {String}   buildId          The ID of the build to stream
+     * @param  {Object}   stream           The reply object to stream to
+     */
+    stream(buildId, stream) {
```
Comment (FenrirUnbound) [inline lib/buildmodel.js]: If `stream` is supposed to be the `reply` object, would it be more clear to keep the names the same?

**Comment by tkyi** [comment]
👍 
giphy

**Comment by d2lam** [comment]
👍

**Pull request merged**